src/parser/elements_parser.ts

```diff
@@ -139,16 +143,32 @@ export class ElementsParser {
     ) as Element[];
     if (clipPathElements) {
       const objTransformInv = invertTransform(obj.calcTransformMatrix());
-      // move the clipPath tag as sibling to the real element that is using it
-      const clipPathTag = clipPathElements[0].parentElement;
+      const clipPathTag = clipPathElements[0].parentElement!;
       let clipPathOwner = usingElement;
       while (
         clipPathOwner.parentElement &&
         clipPathOwner.getAttribute('clip-path') !== obj.clipPath
       ) {
         clipPathOwner = clipPathOwner.parentElement;
       }
+      // move the clipPath tag as sibling to the real element that is using it
       clipPathOwner.parentElement!.appendChild(clipPathTag!);
+
+      // this multiplication order could be opposite.
+      // but i don't have an svg to test it
+      // at the first SVG that has a transform on both places and is misplaced
+      // try to invert this multiplication order
+      const finalTransform = parseTransformAttribute(
+        `${clipPathOwner.getAttribute('transform') || ''} ${
+          clipPathTag.getAttribute('originalTransform') || ''
+        }`
+      );
+
+      clipPathTag.setAttribute(
+        'transform',
+        `matrix(${finalTransform.join(',')})`
+      );
```

Comment:
@mikehollibaugh this is part 1 of the fix. You can find a function similar to this in 4.x